src/models/events/MessageEvent.ts

```diff
@@ -251,4 +251,11 @@ export class MessageEvent<T extends MessageEventContent> extends RoomEvent<T> {
         if (!body && body !== "") throw new EventRedactedError("missing body");
         return body;
     }
+
+    /**
+     * The `external_url` of the message, if it exists
+     */
+    public get externalUrl(): string | undefined {
+        return this.content.external_url || undefined;
```

Comment:
Likewise, this doesn't need the boolean logic
```suggestion
        return this.content.external_url;
```

Reply:
The idea was to filter out invalid things of setting it to e.g. `null` that it still appears correctly as `undefined` as per the type definition of the getter.

Reply:
Falsey values are fine here, I think.